### PR TITLE
Use Monolog:SyslogHandler syslog default facility

### PR DESCRIPTION
**Pull request by mwithheld**
### Description:

This is a further fix to #17764, fixing the Monolog:SyslogHandler create() call to use its constructor-default facility rather than manually setting it to 'syslog' incorrectly.

### CODE REVIEW

**Comment by diosmosis** [comment]
@mwithheld I think you'll need to merge these changes w/ 4.x-dev to remove the conflicts, something like:

```
git checkout 4.x-dev
git pull upstream 4.x-dev # upstream should be the matomo-org/matomo repo
git checkout patch-2
git merge 4.x-dev
```

File: plugins/Monolog/config/config.php [hunk 1]
```diff
@@ -105,7 +105,8 @@
         ->method('setFormatter', DI\get('log.lineMessageFormatter.file')),
 
     '\Monolog\Handler\SyslogHandler' => DI\create()
```
Comment (diosmosis) [inline plugins/Monolog/config/config.php]:
```suggestion
    '\Monolog\Handler\SyslogHandler' => DI\autowire()
```

File: plugins/Monolog/config/config.php [hunk 2]
```diff
@@ -105,7 +105,8 @@
...
-        ->constructor(DI\get('log.syslog.ident'), 'syslog', DI\get('log.level.syslog'))
+        ->constructorParameter('ident' => DI\get('log.syslog.ident'))
```
Comment (diosmosis) [inline plugins/Monolog/config/config.php]:
```suggestion
        ->constructorParameter('ident', DI\get('log.syslog.ident'))
```

File: plugins/Monolog/config/config.php [hunk 3]
```diff
@@ -105,7 +105,8 @@
...
+        ->constructorParameter('level' => DI\get('log.level.syslog'))
```
Comment (diosmosis) [inline plugins/Monolog/config/config.php]:
```suggestion
        ->constructorParameter('level', DI\get('log.level.syslog'))
```

**Comment by diosmosis** [comment]
@mwithheld left a couple suggestions, after those I should be able to merge it.

**Comment by diosmosis** [comment]
Hi @mwithheld, sorry for the late reply, it seems like you added some commits for another PR into this one (the one dealing w/ NotSupportedBrowserException).

**Comment by mwithheld** [comment]
Good catch - I've reset the branch to before the incorrect ones, force-pushed the branch, and merged updates from the 4.x-dev branch.  Hopefully that is clean now.

**Comment by diosmosis** [comment]
Thanks for fixing up the PR @mwithheld! It's now merged.

**Pull request merged**